scripts/install

```diff
@@ -4,6 +4,7 @@ set -e # Exit on error
 # Retrieve arguments
 domain=$1
 is_public=$2
+analytics=$YNH_APP_ARG_ANALYTICS
```

Comment:
It may be worth saving the user preference in Yunohost app settings (using `sudo yunohost app setting …`). This way, if we ever need to re-regenerate the config file from scratch, we'll know what the user requested then.